### DISCUSSION TITLE
Introductions - welcome to batch5.buidlguidl.com!

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch5.buidlguidl.com#1] [Announcements]
Hey everyone! It's Eda from BuidlGuidl. I'm a developer and been part of BG for almost 2 years now! I hope that you'll have fun during this batch and learn a lot. See you on telegram :)

**Comment by carletex** [comment]
Hey all!! 

My name is Carlos, I've been part of the BuidlGuidl since 2021. I've been a software developer for more than 10 years and went _onchain_ three years ago.

During this time I've built things like the [SpeedRunEthereum](https://speedrunethereum.com/) platform, the [BuidlGuidl App](https://app.buidlguidl.com/), [ABI Ninja](https://abi.ninja/) & [Scaffold-ETH 2](https://github.com/scaffold-eth/scaffold-eth-2). Inside the BuidlGuild, I also run the [Sand Garden](https://sandgarden.buidlguidl.com/).

I love learning and building things 👾

Alongside other BG members, I'll be helping you all to go through this batch. Hope you all have fun!

**Comment by devcer** [comment]
Hey all,

I'm Santosh from Hyderabad, India. I have been a Web 2 Developer for over 9 years. Attended the BuidlGuidl workshops at Fossasia 2024 and now I aspire to upskill myself to become a Web3 Developer. Excited to be a part of this cohort.

**Comment by temitopeohassan** [comment]
Hello everyone 
I'm Temitope Hassan, from Nigeria. I've been working on Web3 for about 3 years. Recently got to know about Scaffold Eth from Kevin Jones' workshop during EthDenver. Started SpeedRunEthereum to improve understanding of the package. Found the Buidl Guild in the process

**Comment by zapaz** [comment]
hi all from France,  previously web2 entrepeneur, entered web3 4 years ago, developing a NFTs Factory @[Kredeum](https://www.kredeum.com). With expertise in Solidity, Svelte,  and Certora FV

**Comment by 0xAzerton** [comment]
Hi I am Edgar, dived into blockchain and crypto 8 years ago and specialized myself in DeFi. I co-created the first borrowing aggregator on Ethereum at ethGlobal hackathon the past cycle and I am now focused on improving my coding skills with the goal to collaborate on different projects in the future. Very happy to join buidlguild as I used Scaffold v1 to build some poc and I find the batch formula to be perfect for me to have hands in dirt with other people and not only on theory.

**Comment by SahilKharya** [comment]
Hey Everyone, Sahil here.
I graduated last December with Masters in Software Engineering and currently in USA. I have worked as a Full-stack Developer and now working in a small startup. Have worked on some side web3 projects and would love to involve more in the web3 space.
